lib/time.py

```diff
@@ -23,3 +55,48 @@ def sleep(secs):
 
 def time():
   return float(Now().UnixNano()) / Second
+
+
+_strftime_directive_map = {
+    'Y': '2006', 'y': '06', 'b': 'Jan', 'B': 'January', 'm': '01', 'd': '02',
```

Comment:
Would you mind reformatting this dict to be one directive per line? Although it takes up more space, I think it'll be easier to read. Also, I would put the keys in alphabetical order.